src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = price * days;
+
+    return result - 50;
+  } else if (days >= 3) {
+    result = price * days;
+
+    return result - 20;
+  }
```

Comment:
According to checklist item 5, you should avoid using 'else' after 'if' with 'return'. The code after 'return' won't be executed anyway, so you can simplify the structure by removing 'else'.